app/services/blocks-gallery.js

```diff
@@ -164,10 +164,22 @@ export default Service.extend({
     return bloque;
   },
 
+
+  /*
+   * Devuelve un objeto con un atributo 'string'. Ej: { string: "Al empezar a ejecutar" }
+   */
+  traduccion(id) {
+    return this.intl.t(`blocks.${id}`);
+  },
+
+  stringTraducidoDe(id) {
+    return this.traduccion(id).toString();
+  },
```

Comment:
Ufff después reviso bien, pero mientras comento: Olvidé decirles @dlopezalvaz @ezequielPereyra que lean las contribution guidelines:
https://github.com/Program-AR/pilas-bloques/blob/develop/CONTRIBUTING.md

Todo código de Pilas Bloques debe estar en Inglés. ¿`this.t` y ya?

- Re banco escribir `blocks.` en un solo lugar :+1: 
- Pero la otra función no me termina de quedar claro que sea necesaria. No sé, quizás está bueno ver dónde querés un string y dónde no, no me queda clara la ventaja de delegar .

Reply:
Ahora que leí las guidelines todo tiene sentido (? Vi los dos lenguajes y me tiré por uno jajaja. Ahora lo cambio a inglés. 

La ventaja de delegar es que no tengo que escribir `.toString()` en varios lados (?. Hice dos funciones porque a veces necesitas una y a veces la otra. Pero también entiendo que la segunda no aporta mucho, fue sólo por una cuestión de comodidad. La puedo eliminar y escribir `this.traduccion(id).toString()` en donde se necesite.